Testing Your Seed.md

```diff
@@ -0,0 +1,34 @@
+# Testing Your Seed
+
+Once you have made the backup of a Seed, this Seed can be used to access your Skywallet. 
+
+**NOTE - The Seed generated for a Skywallet can only be used with a Skywallet. You cannot use this Seed in the Desktop Wallet.**
+
+The **Confirm Seed** button in the Hardware Wallet window of your Desktop Wallet can help you in such situations to verify whether a Seed belongs to a particular Skywallet.
+
+The "Confirm Seed" button will not be available when you are using your Skywallet for the first time. Instead you will be having the button **Create Backup** and once you have clicked on the **Create Backup** button on the Hardware Wallet window and completed the process of the Seed backup, "Create Backup" button will be replaced by the "Confirm Seed" button.
+
+<Two Screenshots where the first one is the Hardware Wallet window highlighting Create Backup button and the second one of the same window but highlighting the button confirm seed>
+
+In order to check whether a Seed belongs to a particular Skywallet, you have to connect your Skywallet first. 
+
+***If you are connecting the Skywallet for the first time, please read the following manuals - [Initialization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Initialize-the-wallet-setting-up) and [Personalization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Getting-to-know-the-wallet) to understand how to initialize and personalize your Skywallet for its first time use.***
+
+If you are seeing the "Confirm Seed" button on your Hardware Wallet window, then it means that you have already taken the backup of the Seed.
+
+After clicking on the "Confirm Seed" button the Skywallet will ask you to enter the different words in your Seed.
+
+**However, the sequence of these word positions will be in a random order.**
+
+<Picture of the Skywallet asking the user to enter a word on a particular position and a screenshot of the Desktop Wallet with the window showing the simultaneous message to enter the particular word at that position>
+
+<Optional picture may be to explain the random sequence by reusing the last image and screenshot along with the next message displayed by the Skywallet to enter another word in another position and its simultaneous Desktop Wallet screenshot>
+
+Along with asking you to enter the words of the Seed at a particular position, **the Skywallet may ask you to enter a random word which is not part of the Seed.**
```

Comment:
Please say random words. Otherwise the user might think multiple random words are an error or compromised wallet.